tasks/main.yml

```
@@ -18,17 +18,18 @@
       vars:
         ansible_become: false
       run_once: true
-      when: not is_virtualenv or is_virtualenv == None
+      when: not virtualenv or virtualenv == None
```

```suggestion
      when: virtualenv == ""
```
...since it will always be defined in the role defaults.